packages/runner/src/builtins/wish.ts

```diff
@@ -47,14 +46,25 @@ function parseWishTarget(target: string): ParsedWishTarget | undefined {
   if (trimmed.startsWith("#")) {
     const rest = trimmed.slice(1);
     const segments = rest.split("/").filter((segment) => segment.length > 0);
-    const baseSegment = segments.shift();
-    if (!baseSegment) return undefined;
-    const key = `#${baseSegment}`;
-    return { key, path: segments };
+
```

Comment:
Could have a `WishKey` type using these sigils (`#`, `/`) in `@commontools/api`/with the Wish params that patterns and here in the runtime could use

```ts
type WishKey = `/${string}` | `#${string}`
```

though a bit strange that `#`, `/` and well-known `#allCharms` are all `keys` -- if `#` targets properties on the Default App, why do we manage `#allCharms` in the runtime? almost seems like `#allCharms` is a unique thing, maybe should have a different sigil e.g. `$ALL_CHARMS` (caps for constant-ish)